src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/Main.cs

```diff
@@ -64,7 +64,7 @@ private Result GetResult(UnitConversionResult result)
                 Title = result.ToString(null),
                 IcoPath = _icon_path,
                 Score = 300,
-                SubTitle = string.Format(CultureInfo.CurrentCulture, CopyToClipboard, result.QuantityInfo.Name),
+                SubTitle = (result.UnitNameFrom == "usounce" ? "From US Ounce - " : (result.UnitNameFrom == "imperialounce" ? "From Imperial Ounce - " : string.Empty)) + string.Format(CultureInfo.CurrentCulture, CopyToClipboard, result.QuantityInfo.Name),
```

Comment:
Please add "From {0}" to resex and use it with `string.Fromat()`. In resex we need a comment like `Like "Convert from Unit".`.

And I wondering if we can improve it by having two or three different resex strings containing all subtitle elements and a switch case before `return new Result ...`. The code line looks a bit messy.